articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,487 @@
+
+1. Access and log into the [Azure Portal](https://portal.azure.com).
+
+2. Open the sample application resource group `application-insights-azure-cafe`.
+
+3. From the list of resources, select the `azure-cafe-insights-{SUFFIX}` Application Insights resource.
+
+4. Select **Application map** from the left menu, beneath the **Investigate** heading. Observe the generated Application map.
+
+    ![The Application Insights application map displays.](./media/asp-net-core/application-map.png "Application map")
+
+### Viewing HTTP calls and database SQL command text
+
+1. In the Azure Portal, open the Application Insights resource.
+
+2. Beneath the **Investigate** header on the left menu, select **Performance**.
+
+3. The **Operations** tab contains details of the HTTP calls received by the application. You can also toggle between Server and Browser (client-side) views of data.
+
+    ![The Performance screen of Application Insights displays with the toggle between Server and Browser highlighted along with the list of HTTP calls received by the application.](./media/asp-net-core/server-performance.png "Server performance HTTP calls")
+
+4. Select an Operation from the table, and choose to drill into a sample of the request.
+
+    ![The Performance screen displays with a POST operation selected, the Drill into samples button is highlighted and a sample is selected from the suggested list.](./media/asp-net-core/select-operation-performance.png "Drill into an operation")
+
+5. The End-to-end transaction displays for the selected request. In this case, a review was created including an image, thus it includes calls to Azure Storage, the Language Service (for sentiment analysis), as well as database calls into SQL Azure to persist the review. In this example, the first selected Event displays information relative to the HTTP POST call.
+
+    ![The End-to-end transaction displays with the HTTP Post call selected.](./media/asp-net-core/e2e-http-call.png "HTTP POST details")
+
+6. Select a SQL item to review the SQL command text issued to the database.
+
+    ![The End-to-end transaction displays with SQL command details.](./media/asp-net-core/e2e-db-call.png "SQL Command text details")
+
+7. Optionally select Dependency (outgoing) requests to Azure Storage or the Language Service.
+
+8. Return to the **Performance** screen, and select the **Dependencies** tab to investigate calls into external resources. Notice the Operations table includes calls into Sentiment Analysis, Blob Storage, and Azure SQL.
+
+    ![The Performance screen displays with the Dependencies tab selected and the Operations table highlighted.](./media/asp-net-core/performance-dependencies.png "Dependency Operations")
+
+### Sampling
+
+The Application Insights SDK for ASP.NET Core supports both fixed-rate and adaptive sampling. By default, adaptive sampling is enabled. You can disable or customize the sampling behavior.
+
+#### Disabling adaptive sampling
```

Comment:
My suggestion is to remove this whole adaptive sampling section. Its not required to folow this tutorial, and sampling is documented elsewhere.

Reply:
Fixed.